tools/rna_tools/locarna/locarna_pairwise.xml

```diff
@@ -0,0 +1,284 @@
+    #end if
+
+    '$inputA'
+    '$inputB'
+    
+    --width 60
+    
+    ## -------------------- alignment mode and specific options
+
+    #if str($alignment_mode.alignment_mode_selector) == "global_locarna"
+        #if str($alignment_mode.free_endgaps) != "----"
+            --free-endgaps $alignment_mode.free_endgaps
+        #end if
+    #else if str($alignment_mode.alignment_mode_selector) == "local_locarna"
+        --sequ-local on
+    #else if str($alignment_mode.alignment_mode_selector) == "structure_local_locarna"
+        --struct-local on
+    #else if str($alignment_mode.alignment_mode_selector) == "structure_sequence_local_locarna"
+        --sequ-local on
+        --struct-local on
+    #end if
+
+    ## -------------------- scoring parameters
+    
+    --indel $Scoring.indel
+    --indel-opening $Scoring.indel_opening
+    --struct-weight $Scoring.struct_weight
+    --tau $Scoring.tau
+    
+    #if str($Scoring.sequence_score.sequence_score_selector) == "match"
+        --match $Scoring.sequence_score.match
+        --mismatch $Scoring.sequence_score.mismatch
+    #else if str($Scoring.sequence_score.sequence_score_selector) == "ribosum"
+        --use-ribosum true
+    #else if str($Scoring.sequence_score.sequence_score_selector) == "ribofit"
+        --ribofit true
+    #end if
+
+    ## -------------------- folding parameters
+
+    #if float($Folding.rnafold_temperature) != 37.0
+        --rnafold-temperature $Folding.rnafold_temperature
+    #end if
+
+    $Folding.alifold_consensus_dp
+    --consensus-structure $Folding.consensus_structure
+
+    ## -------------------- heuristic parameters
+    
+    -p $Heuristics.min_prob
+    --max-diff-am $Heuristics.max_diff_am
+    --max-diff $Heuristics.max_diff
+    --max-diff-at-am $Heuristics.max_diff_at_am
+    
+    #if float($Heuristics.max_bps_length_ratio) != 0.0
+        --max-bps-length-ratio $Heuristics.max_bps_length_ratio
+    #end if
+    
+    ## -------------------- other parameters
+        
+    $Constraint.lonely_pairs
+
+    #if $Constraint.maxBPspan != -1
+        --maxBPspan $Constraint.maxBPspan
+    #end if
+
+    $Constraint.ignore_constraints
+
+    ## -------------------- output
+    
+    #if 'stockholm' in str($outputs).split(",")
+        --stockholm '$stockholm'
+    #end if
+    #if 'clustal' in str($outputs).split(",")
+        --clustal '$clustal'
+    #end if
+    #if 'clustal_strict' in str($outputs).split(",") and (not 'clustal' in str($outputs).split(","))
+        --clustal '$clustal_strict'
+    #end if
+    #if 'pp' in str($outputs).split(",")
+        --pp '$pp'
+    #end if
+    
+    $stdout_verbosity
+
+    #if str($stdout_verbosity) != "--quiet":
+        > '$stdout'
+    #end if
+
+    #if 'clustal' in str($outputs).split(",") and 'clustal_strict' in str($outputs).split(",")
+        && cp $clustal $clustal_strict
+    #end if
+
+    #if 'clustal_strict' in str($outputs).split(",")
+        && sed -i '/\^#/d' '$clustal_strict'
```

Comment:
This will not work on OSX. `-i` needs to be replaces by `-i.bak` or you can strictly depend on `sed`